core/services/job/orm.go

```diff
@@ -431,27 +442,36 @@ func loadDynamicConfigVars(cfg Config, os OffchainReportingOracleSpec) *Offchain
 	}
 }
 
+func (o *orm) FindJobTx(id int32) (Job, error) {
```

Comment:
There are a few top level endpoints which just want to lookup a job and don't have a TxManager nor direct access to a db to create one.